CHANGELOG

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Upcoming]
 
+## [0.17.0] — 2019-06-25
```

Comment:
<!--
  1 failure:  Illegal date stri...
  0 warning: 
  
  1 markdown notices
  DangerID: danger-id-Danger;
  File: CHANGELOG;
  Line: 9;
-->
- :no_entry_sign: Illegal date string in CHANGELOG!


```suggestion
## [0.0.0] - 2019-07-25
```